code/modules/surgery/surgery_step.dm

```diff
@@ -72,6 +72,12 @@
 #define SURGERY_SPEED_MORBID_CURIOSITY 0.7
 ///Modifier given to patients with TRAIT_ANALGESIA
 #define SURGERY_SPEED_TRAIT_ANALGESIA 0.8
+///The penalty for trying to do a surgery as an untrained person
+#define SURGERY_SPEED_TRAIT_UNTRAINED 1.3
+///As above but the inverse; only applied in un-ideal conditions
+#define SURGERY_SPEED_TRAIT_TRAINED 0.7
+///Job trained in surgery: medbay for medicine, science for dissection, cyborgs and AI
+#define JOBS_TRAINED_IN_SURGERY list(/datum/job/paramedic, /datum/job/doctor, /datum/job/chief_medical_officer, /datum/job/scientist, /datum/job/research_director, /datum/job/cyborg, /datum/job/ai)
```

Comment:
- Weird that this includes the RD and the AI and not roboticists. 
- We have (largely) been moving away from bonuses tied directly to job datums, outside of meme ones like bar drinks. Most similar bonuses have been moved to liver traits and skill chips. 
   - Point being: I think you should find an alternative method of granting these jobs a skill bonus. Maybe add it to the entrails reader skillchip (and let cyborgs have it naturally?)

Reply:
I see; I wasn't aware of the move away from job features toward gear instead. I'll dig around and make the changes.

Regarding Roboticist: I forgor

Reply:
i added surgical training to the entrails skillchip; this means the coroner will also have surgical training, but c'est a la vie for now